specs/CardanoShellSpec.lhs

```diff
@@ -207,6 +207,9 @@ When the \textit{Daedalus} calls and starts the \textit{Node}, it also opens up
 First, the \textit{Node} sends the message \textbf{Started} back to the \textit{Daedalus} to inform him that the communication can begin.
 After that, \textit{Daedalus} sends the message \textbf{QueryPort} to the \textit{Node}, and the \textit{Node} responds with the free port it found using \textbf{ReplyPort PORTNUM} that is going to be used for starting the HTTP "server" serving the \textit{JSON API} which they can then use to communicate further.\\
 
+Not only does the \textit{Node} responds to the message, but it can also perform some IO actions depending on the message sent from the client.
+For instance, you can have the node to start an new process when the client sends \textbf{Started} or have it so that node will kill its thread/process when it responds with \textbf{ReplyPort PORTNUM}.\\
```

Comment:
`start an new process` - `start a new process`.
"you can have the node to start an new process when the client sends \textbf{Started} or have it so that node will kill its thread/process when it " - what does this mean? Maybe an example where the response is an action called "FetchInfo SystemInfo" where the client needs to fetch the client info and return it to the server. That would require `IO`.